src/view/com/composer/text-input/web/Autocomplete.tsx

```diff
@@ -151,7 +156,7 @@ const MentionList = forwardRef<MentionListRef, SuggestionProps>(
       <div className="items">
         <View style={[pal.borderDark, pal.view, styles.container]}>
           {items.length > 0 ? (
-            items.map((item, index) => {
+            items.sort(sortProfiles).map((item, index) => {
```

Comment:
`sort` is mutative so we can't do it here.

ideally, if the server response isn't appropriately sorted but we _want_ it to be sorted, we'd place this logic closest to where the response is being parsed — i.e. in the query. mind doing that?